plugins/auth/ref10/pow22523.h

```diff
@@ -50,7 +50,7 @@
 /* qhasm: z2 = z1^2^1 */
 /* asm 1: fe_sq(>z2=fe#1,<z1=fe#11); for (i = 1;i < 1;++i) fe_sq(>z2=fe#1,>z2=fe#1); */
 /* asm 2: fe_sq(>z2=t0,<z1=z); for (i = 1;i < 1;++i) fe_sq(>z2=t0,>z2=t0); */
-fe_sq(t0,z); for (i = 1;i < 1;++i) fe_sq(t0,t0);
+/* fe_sq(t0,z); for (i = 1;i < 1;++i) fe_sq(t0,t0); */
```

Comment:
Only for loop should be removed/commented out

Reply:
Thanks, should I change it in the PR? Because I see that you've already made an commit with correct one.